fenix/app/src/androidTest/java/org/mozilla/fenix/ui/LoginsTest.kt

```diff
@@ -52,6 +48,42 @@ class LoginsTest {
         mockWebServer.shutdown()
     }
 
+    // Tests only for initial state without signing in.
+    // For tests after singing in, see SyncIntegration test suite
+    @Test
+    fun loginsAndPasswordsSettingsItemsTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+            // Necessary to scroll a little bit for all screen sizes
+            TestHelper.scrollToElementByText("Logins and passwords")
+        }.openLoginsAndPasswordSubMenu {
+            verifyDefaultView()
+            verifyDefaultValueAutofillLogins(TestHelper.appContext)
+            verifyDefaultValueExceptions()
+        }.openSavedLogins {
+            verifySecurityPromptForLogins()
+            tapSetupLater()
+            // Verify that logins list is empty
+            // Issue #7272 nothing is shown
```

Comment:
Is this now fixed? I checked https://github.com/mozilla-mobile/fenix/issues/7272 and it seems fixed. If not, we should be linking to Bugzilla now.

Reply:
I'll check it out